scripts/Vagrantfile

```diff
@@ -0,0 +1,35 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+# Author: David Manouchehri
```

Comment:
can you remove `Author` - things in the repo are collective responsibility

Reply:
You're welcome to remove it.